### ISSUE TITLE
Undefined Index --> Attribute overview -> Sharing -> no sharing group selected

### ISSUE THREAD

**Issue by jaegeral** [MISP/MISP#2116]
```
Notice (8): Undefined index: name [APP/View/Elements/eventattribute.ctp, line 424]
```

*If you would like to report a bug, please fill the template bellow*

### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | 
| OS version (client)       | 
| PHP version               | 
| MISP version / git hash   | 2.4.70
| Browser                   | 

### Expected behavior
If you have an event with an attribute and select "sharing group" as sharing right out of the attribute overview table, you can select sharing group, without specifiying the SG itself. But UI is fine with it.

So basically the form to select the sharing group is missing (if it is not possible in that ui it should not be able to select the Sharing group at all)

### Actual behavior

```
Notice (8): Undefined index: name [APP/View/Elements/eventattribute.ctp, line 424]
```
